bindings/py/tests/nupic_random_test.py

```diff
@@ -201,34 +202,41 @@ def testShuffleEmpty(self):
     self.assertEqual(arr.size, 0)
 
 
-  @pytest.mark.skip(reason="Does not throw...another PR")
   def testShuffleEmpty2(self):
     r = Random(42)
-    arr = numpy.zeros([2, 2], dtype="uint32")
+    arr = numpy.zeros([2, 2], dtype="uint32")#2x2 array dimension
 
     self.assertRaises(ValueError, r.shuffle, arr)
 
 
-  @pytest.mark.skip(reason="Does not throw...another PR")
   def testShuffleBadDtype(self):
     r = Random(42)
     arr = numpy.array([1, 2, 3, 4], dtype="int64")
 
     self.assertRaises(ValueError, r.shuffle, arr)
 
 
-  @pytest.mark.skip(reason="not equal...another PR")
   def testEquals(self):
     r1 = Random(42)
     v1 = r1.getReal64()
     i1 = r1.getUInt32()
+    s1 = r1.getSeed()
+    m1 = r1.min()
+    n1 = r1.max()
+    
     r2 = Random(42)
     v2 = r2.getReal64()
     i2 = r2.getUInt32()
+    s2 = r2.getSeed()
+    m2 = r2.min()
+    n2 = r2.max()
+    
     self.assertEqual(v1, v2)
-    self.assertEqual(r1, r2)
```

Comment:
why is the (Random) r comparison replaced by s,m,n? (and is it adequate?)

Reply:
We cant expect in python that two instances of Random() will be equal, like r1=r2. 
In C++ code of Random class there is operator==() but i dont know how to bind this to python. So i just added more inner variable comparism..  Any thoughts how to solve in better way?

Reply:
> cant expect in python that two instances of Random() will be equal, like r1=r2.
In C++ code of Random class there is operator==()  

as you're saying, in c++ the operator works well, so we should provide it to python. 

https://developer.lsst.io/v/DM-9089/coding/python_wrappers_for_cpp_with_pybind11.html#wrapping-operators

I think `def __eq__(const Random& self, const Random& other)` to something like `return self==other; `

Reply:
nice, thx